### PR TITLE
fix: avg rates should be ints

### PR DESCRIPTION
**Pull request by chasers**
Noticed long floats on staging. This fixes it.

### CODE REVIEW

File: lib/logflare/source/rate_counter_server.ex [hunk 1]
```diff
@@ -268,14 +268,28 @@ defmodule Logflare.Source.RateCounterServer do
...
+  """
+
+  @spec stats([integer()]) :: %{avg: integer(), sum: integer()}
   def stats(xs) when is_list(xs) do
     {total, count} =
       Enum.reduce(xs, {0, 0}, fn v, {total, count} ->
         {total + v, count + 1}
       end)
 
+    avg = Kernel.ceil(total / count)
```
Comment (hauleth) [inline lib/logflare/source/rate_counter_server.ex:289]: Ceil there is correct function? I would need to run benchmark whether this or `div(total + count - 1, count)` is faster.

Comment (Ziinc) [inline lib/logflare/source/rate_counter_server.ex:289]: Need some  checks for div by zero for either numerator denominator

Comment (Ziinc) [reply lib/logflare/source/rate_counter_server.ex:289]: I think it is for rounding

**Review by Ziinc — CHANGES_REQUESTED** [review]
Needs an if check for division

File: lib/logflare/source/rate_counter_server.ex [hunk 2]
```diff
@@ -268,14 +268,28 @@ defmodule Logflare.Source.RateCounterServer do
     Sources.Counters.get_inserts(source_id)
   end
 
+  @doc """
+  Takes a list of integers representing per second counts of events
+  of a source and returns the `avg` and `sum` of those counts.
+
+  ## Examples
+
+      iex> Logflare.Source.RateCounterServer.stats([12,5,30,5,1000])
```
Comment (Ziinc) [inline lib/logflare/source/rate_counter_server.ex:277]: Adding in a case for empty list should cover the div by zero issue

File: lib/logflare/source/rate_counter_server.ex (hunk 1 above)
Comment (chasers) [reply lib/logflare/source/rate_counter_server.ex:289]: Ya it’s for rounding. You can take this over to get it in the latest version.

Comment (chasers) [reply lib/logflare/source/rate_counter_server.ex:289]: Long floats were on the dashboard.

Comment (hauleth) [reply lib/logflare/source/rate_counter_server.ex]: @Ziinc I know that this is for rounding. Just my point is whether `trunc/1` (faster) or `round/1` (more correct) would not work better.

**Pull request merged**